Changing fsck to recover the corrupt/missing files (#250)
HPInc/ml-git: docs/mlgit_commands.md

Usage: ml-git datasets fsck [OPTIONS]

Options:
  --fix-workspace  Use this option to repair files identified as corrupt in
                   the entity workspace.
  --full           Show the list of corrupted files.
  --verbose        Debug mode
```

Example:
```
ml-git datasets fsck
```

This command will walk through the internal ml-git directories (index & local repository) and will check the integrity and presence of all blobs under its management.

This command will basically try to:

* Detects any chunk/blob that are corrupted or missing in the internal ml-git directory (.ml-git/{entity-type}/objects)
* Attempts to fetch files detected as corrupt or missing from storage
* Checks the integrity of files mounted in the entities workspace

> * In fix-workspace mode, try to repair corrupted files founded in the entities workspace

Author: Additional text has been added to explain. [14e766f](https://github.com/HPInc/ml-git/pull/250/commits/14e766f3396b6fbe5c4e0a6a302a844c88ef3e3c)

Usage: ml-git datasets fsck [OPTIONS]

Options:

>   --fix-workspace  Use this option to repair files identified as corrupt in

Reviewer: ```suggestion
  --fix-workspace  Use this option to repair files identified as corrupted in
```

Author: This and other suggestions were applied in [ca1ee90](https://github.com/HPInc/ml-git/pull/250/commits/ca1ee902e4c995dacd0d80b090c52adc39601444)

                   the entity workspace.
  --full           Show the list of corrupted files.
  --verbose        Debug mode
```

Example:
```
ml-git datasets fsck
```

> This command will walk through the internal ml-git directories (index & local repository) and will check the integrity and presence of all blobs under its management.

Reviewer: ```suggestion
This command will walk through the internal ml-git directories (index & local repository) and check the presence and integrity of all file blobs under its management.
```

This command will basically try to:

* Detects any chunk/blob that is corrupted or missing in the internal ml-git directory (.ml-git/{entity-type}/objects)

> * Attempts to fetch files detected as corrupt or missing from storage

Reviewer: ```suggestion
* Fetch files detected as corrupt or missing from storage
```

> * Checks the integrity of files mounted in the entities workspace

Reviewer: ```suggestion
* Check the integrity of files mounted in the entities workspace
```

> * In fix-workspace mode, try to repair corrupted files founded in the entities workspace. A file in the entities workspace is considered corrupted based on the business rule defined by the mutability of the entity. 

Reviewer: ```suggestion
* In fix-workspace mode, repair corrupted files found in the entities workspace. A file in the entities workspace is considered 'corrupted' based on the business rule defined by the mutability of the entity. 
```
